frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -59,26 +65,56 @@ export const AppearanceSettings = ({
   );
   const utmTags = `?utm_source=${plan}&utm_media=static-embed-settings-appearance`;
 
-  const fontControlLabelId = useUniqueId("display-option");
   const downloadDataId = useUniqueId("download-data");
 
   return (
     <>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Customizing your embed’s appearance`}
-      >
-        <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${(
-          <ExternalLink
-            key="doc"
-            href={`${docsUrl}${utmTags}#customizing-the-appearance-of-static-embeds`}
-          >{t`documentation`}</ExternalLink>
-        )} for more.`}</Text>
-      </StaticEmbedSetupPaneSettingsContentSection>
-      <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Playing with appearance options`}
-        mt="2rem"
+        title={t`Customizing look and feel`}
       >
         <Stack spacing="1rem">
+          <Text>{jt`These options require changing the server code. You can play around with and preview the options here. Check out the ${(
```

Comment:
Slightly modify the markup to match the parameter tab.

https://github.com/metabase/metabase/blob/bdae7827e4cba3114420f41307160d324eec5c5b/frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/ParametersSettings.tsx#L63-L66

So in the section, we start off with
1. Stack (This separates the description and settings with a space)
2. Text (description)
3. A list of settings